WalletWasabi.Tests/UnitTests/WabiSabi/Backend/CoinVerifierTests.cs

```diff
@@ -80,7 +100,26 @@ public async Task CanFilterNaughtyUtxoTestAsync()
 
 		List<Coin> generatedCoins = GenerateCoins(10);
 
-		await foreach (var item in coinVerifier.VerifyCoinsAsync(generatedCoins, CancellationToken.None).ConfigureAwait(false))
+		uint256 roundId = uint256.One;
```

Comment:
According to this https://stackoverflow.com/a/15024485

if we want to keep this test as it is, we need to duplicate 
```
.ReturnsAsync(() =>
	{
		string content = GenerateCleanJsonReport();
		HttpResponseMessage response = new(System.Net.HttpStatusCode.OK);
		response.Content = new StringContent(content);
		return response;
	});
```
block 8 more times.

Maybe it would be good to only generate 5 coins, so we don't need to duplicate so much code.

Reply:
ACK